index.html

```diff
@@ -0,0 +1,285 @@
+			<p>So the future of the cosmos is perhaps bright and fecund, but do not bother asking about going any deeper into the past.</p>
+			<p>We might never know what happened before inflation, at the very beginning, because inflation erases everything that came before it. All the chaos and randomness of the primordial moment are swept away, forever out of our view.</p>
+			<p>“If you trace your cosmic roots,” said Abraham Loeb, a Harvard-Smithsonian astronomer who was not part of the team, “you wind up at inflation.”</p>
+			<p id='ending'>A version of this article appears in print on March 18, 2014, on Page A1 of the New York edition with the headline: Space Ripples Reveal Big Bang’s Smoking Gun. Order Reprints | Today’s Paper | Subscribe</p>
+		</div>
+		<button>READ 615 COMMENTS</button>
+	</div>
+	<div class='suggestSection'>
+		<h3>More Space and Cosmos</h3>
+		<div class='box'>
+			<img src="pictures/img2.jpg">
+			<h6>Craig Bailey/Florida Today, via Associated Press</h6>
+			<p>Snowman-like Photo of Ultima Thule Sent Home by NASA'S New Horizons Spaceraft</p>
+			<h6 class = "date">Jan 6</h6>
+		</div>
+		<div class='box'>
+			
+			<img src="pictures/img3.jpg">
+			<h6>ASA, ESA, and G. Bacon (STScI)</h6>
+			<p>'New Chapter' in Space Exploaration  as China Reaches Far Side of the Moon</p>
+			<h6 class = "date">Oct 17</h6>
+		</div>
+		<div class='box'>
+			
+			<img src="pictures/img4.jpg">
+			<h6>China National Space Administration</h6>
+			<p>Rocket Launches, Trips to the Moon and More Space and Astronomy Events in 2019</p>
+			<h6 class = "date">Sep 9</h6>
+		</div>
+		<div class='box'>
+			
+			<img src="pictures/img5.jpg">
+			<h6>NASA</h6>
+			<p>A Journey Into the Solar System’s Outer Reaches, Seeking New Worlds to Explore</p>
+			<h6 class = "date">Jul 6</h6>
+		</div>
+		<div class='box'>
+			
+			<img src="pictures/img6.jpg">
+			<h6>NASA</h6>
+			<p>NASA Apollo 8’s Earthrise: The Shot Seen Round the World</p>
+			<h6 class = "date">May 11</h6>
+		</div>
+
+
+
+
+		</div>
+		<div id='sections'>
+		<h3>Most Popular</h3>
+		<ul>
+			<li>Time’s Up Shows Up on Pins and Bracelets at the Golden Globes</li>
+			<li>What to Cook Right Now</li>
+			<li>A New Syrup for Your Bar Cart</li>
+			<li>Daryl Dragon, of the Captain and Tennille Pop Duo, Dies at 76</li>
+			<li>Chelsea Signs Christian Pulisic for $73 Million</li>
+			<li>At Hotels, Space That’s Like the Office, ‘but Cooler’</li>
+			<li>Britney Spears Announces ‘Indefinite Work Hiatus,’ Cancels Las Vegas Residency</li>
+			<li>Hard Feelings in San Antonio as Kawhi Leonard Returns</li>
+			<li>TA California Dream for Paid Leave Has an Old Problem: How to Pay for It</li>
+		</ul>
+	</div>
+		<div class='suggestSection' id='editorsPick'>
```

Comment:
I'm not crazy about mixing id and class. The idea behind the id is to target an element that it's going to behave in a particular way. The idea behind classes is to set a style you are going to use on multiple occasions.